rapid_pro/docker-run-synchronise-contacts.sh

```diff
@@ -12,6 +12,9 @@ while [[ $# -gt 0 ]]; do
         --force)
             FORCE="--force"
             shift;;
+        --dry-run)
```

Comment:
```suggestion
        --test-run)
```

Reply:
sounds more descriptive

Reply:
I'm not sure about this: the problem with '--test-run' is it might suggest we're in test mode so operating over test data, whereas we're actually running over live data here and just not writing back to Rapid Pro. I might be persuaded to change it to e.g. --simulate or --no-writeback, but from my experience --dry-run is usually what this is called. For example, git calls this dry-run: https://git-scm.com/docs/git-clean#Documentation/git-clean.txt---dry-run

Reply:
Yup that makes sense you can keep --dry-run / --simulate whichever you find more straightforward.